rosalind_hamm.py

```diff
@@ -1,9 +1,3 @@
 def main(data):
-    a, b = data[:-1].split('\n')
-    hamm_dist = 0
-    for i in range(len(a)):
-        if a[i] != b[i]:
-            hamm_dist += 1
-        else:
-            pass
-    return hamm_dist
+    a, b = data.strip().split('\n')
+    return sum([1 for i in range(len(a)) if a[i] != b[i]])
```

Comment:
If you're indexing the elements of a list by iterating over something like `range(len(a))`, there is typically a more pythonic way to do what you're trying to do.

See if you can use `zip` to solve this problem.

```py
>>> zip(a, b)
[('G', 'C'), ('A', 'A'), ('G', 'T'), ('C', 'C'), ('C', 'G'), ('T', 'T'), ('A', 'A'), ('C', 'A'), ('T', 'T'), ('A', 'G'), ('A', 'A'), ('C', 'C'), ('G', 'G'), ('G', 'G'), ('G', 'C'), ('A', 'C'), ('T', 'T')]
```